### DISCUSSION TITLE
how to set path by lua?

### DISCUSSION THREAD

**Discussion by edte** [LunarVim/LunarVim#2816] [Q&A] [answered]
i want to set cpp include path by lua, like ```set path+=/usr/include```,could someone konw how to set it ?

**Comment by kylo252** [comment, marked as answer]
```lua
vim.env.PATH = vim.env.PATH .. "/my/custom/prefix"
```

example https://github.com/williamboman/mason.nvim/blob/3458edb7e020c8e1249b307b084f85ec0230175e/lua/mason/init.lua#L13
